thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/DetectionPipelineScheduler.java

```diff
@@ -82,7 +82,7 @@ public void run() {
 
         try {
           if (scheduler.checkExists(key)) {
-            LOG.info("Detection config  " + key + " is already scheduled.");
+            LOG.info("Detection config  " + key.getName() + " is already scheduled");
```

Comment:
Can you keep both? Having key is easier to debug. E.g, you can quickly look up key in database.

Reply:
This is what is being logged today.
`Detection config  DETECTION.DETECTION_107451840 is already scheduled`
After this change, you will see,
`Detection config  DETECTION_107451840 is already scheduled`